Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -384,4 +356,269 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
     }
 
     return lines;
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _DWriteGetFontFamilyNames() {
+    NSMutableArray<NSString*>* fontFamilyNames = [NSMutableArray<NSString*> array];
+
```

Comment:
I actually spent a bunch of effort converting this from CFArray _to_ NSArray so that this file would be consistent T____T.

I do have versions of the functions I added that are more CF-style. Is our long term goal to make this file more CF-y? (do we care about that?) If so, do I want to also change the existing functions in this file to be more CFy?

---
In reply to: [78221279](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78221279) [](ancestors = 78221279,78220974)